pkg/endpoint/bpf.go

```diff
@@ -225,9 +225,22 @@ func (e *Endpoint) addNewRedirectsFromDesiredPolicy(ingress bool, desiredRedirec
 		Old:  policy.NewMapState(nil),
 	}
 
+	// create or update proxy redirects
 	e.desiredPolicy.UpdateRedirects(ingress,
 		func(l4 *policy.L4Filter) (uint16, bool) {
 			var redirectPort uint16
+
+			proxyID, port := e.proxyID(l4)
```

Comment:
These sorts of things make my head spin :-(. Is this the pod port, the peer port, or the proxy port?

Reply:
renamed as `dstPort` and added this comment above:
```
+				// proxyID() returns also the destination port for the policy,
+				// which may be resolved from a named port
+				proxyID, dstPort := e.proxyID(l4)
```